include/cooperative_perception/temporal_alignment.hpp

```diff
@@ -43,51 +46,52 @@ constexpr Visitor kStatePropagator{ [](auto& object, units::time::second_t time)
 /**
  * @brief Get predicted Detection for specified time
  *
- * @param object DetectionType being predicted
- * @param time Prediction time
- * @return DetectionType whose state corresponds to the specified time
+ * @param[in] detection Detection being predicted
+ * @param[in] time Prediction time
+ * @return DetectionVariant whose state corresponds to the specified time
  */
-auto objectAtTime(const DetectionType& object, units::time::second_t time) -> DetectionType
+auto objectAtTime(const DetectionVariant& detection, units::time::second_t time) -> DetectionVariant
 {
-  DetectionType new_object{ object };
+  DetectionVariant predicted_detection{ detection };
 
-  std::visit(kStatePropagator, new_object, std::variant<units::time::second_t>(time));
+  std::visit(kStatePropagator, predicted_detection, std::variant<units::time::second_t>(time));
 
-  return new_object;
+  return predicted_detection;
 };
 
 /**
  * @brief Get predicted Detections for specified time
  *
- * @param objects List of DetectionTypes being predicted
- * @param time Prediction time
- * @return List of DetectionTypes, each of whose state corresponds to the specified time
+ * @param[in] detections List of DetectionVariants being predicted
+ * @param[in] time Prediction time
+ * @return List of DetectionVariants, each of whose state corresponds to the specified time
  */
-auto objectsAtTime(const DetectionList& objects, units::time::second_t time) -> DetectionList
+auto objectsAtTime(const DetectionVariantList& detections, units::time::second_t time) -> DetectionVariantList
 {
-  DetectionList new_objects{ objects };
+  DetectionVariantList new_detections{ detections };
 
-  std::transform(std::cbegin(new_objects), std::cend(new_objects), std::begin(new_objects),
-                 [time](const DetectionType& object) { return objectAtTime(object, time); });
+  std::transform(std::cbegin(new_detections), std::cend(new_detections), std::begin(new_detections),
+                 [time](const auto& detection) { return objectAtTime(detection, time); });
 
-  return new_objects;
+  return new_detections;
 }
```

Comment:
PR description says we remove these because we are not using it. Should we remove it>?

Reply:
Yes, there were removed in a commit I forgot to push. They are being replaced with other, similar functions in #52